### PR TITLE
Docs: add info about fallback to prerender-error

### PR DESCRIPTION
**Pull request by geritol**

### CODE REVIEW

File: errors/prerender-error.md [hunk 1]
```diff
@@ -9,3 +9,4 @@ While prerendering a page an error occurred. This can occur for many reasons fro
 - Make sure to move any non-pages out of the `pages` folder
 - Check for any code that assumes a prop is available even when it might not be. e.g., have default data for all dynamic pages' props.
 - Check for any out of date modules that you might be relying on
+- Make sure your component handles `fallback` if it is enabled in `getStaticPaths`
```
Comment (timneutkens) [inline errors/prerender-error.md:12]: This should have a link to the relevant documentation for fallback.

**Review by timneutkens — DISMISSED** [review]
Thanks!

**Review by leerob — DISMISSED** [review]

**Comment by leerob** [comment]
I'm not sure why 6 reviews are required here 🤔

**Comment by timneutkens** [comment]
> I'm not sure why 6 reviews are required here 🤔

@leerob this PR was opened against the wrong branch (not canary) hence why it does not allow merging. I've updated the base branch.

**Review by timneutkens — APPROVED** [review]

**Pull request merged**